### PR TITLE
Changed share l10n string to use a generic template for each share menu items.

### LINKED ISSUE #8604: Cleanup share l10n strings to re-use the same string
## Test plan

https://github.com/brave/browser-laptop/pull/12445#issue-285184656

----

See review comment here:
https://github.com/brave/browser-laptop/pull/8517/files/9ce64841f10aa46dcad512307af68966057bca49#diff-985837eca34ef94b3bc9cf3271761dca

It would be cleaner to use the same string for each of these menu items.

### PR DESCRIPTION
**Pull request by sergiorojasa**
Fix #8604

Submitter Checklist:

- [X] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [X] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [X] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

Test Plan:
1. Build from the source: npm run watch
2. Run Brave: npm run start
3. In the URL bar type: https://www.brave.com/
4. On the menu bar click on File.
5. Under File look for Share.
6. Under Share click on one item with share page.

Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/common/commonMenu.js [hunk 1]
```diff
@@ -143,8 +143,10 @@ module.exports.printMenuItem = () => {
 }
 
 module.exports.simpleShareActiveTabMenuItem = (l10nId, type, accelerator) => {
+  const siteName = type.charAt(0).toUpperCase() + type.slice(1)
+  
   return {
-    label: locale.translation(l10nId),
+    label: locale.translation(l10nId).replace(/{{\s*siteName\s*}}/, siteName),
```
Comment (NejcZdovc) [inline app/common/commonMenu.js]: Can you change this one to `locale.translation(l10nId, {siteName: siteName})`? We already have a mechanism for variables in the translations.

Comment (sergiorojasa) [reply app/common/commonMenu.js]: Yes

Comment (sergiorojasa) [reply app/common/commonMenu.js]: I tried your suggestion when I lookup locale.translation it only accepted one argument.

Comment (NejcZdovc) [reply app/common/commonMenu.js]: not sure that I follow, this change is not working for you?

Comment (sergiorojasa) [reply app/common/commonMenu.js]: The change is not working for me.

Comment (NejcZdovc) [reply app/common/commonMenu.js]: ok, let me try it

Comment (NejcZdovc) [reply app/common/commonMenu.js]: ok I see what's going on, will fix it, so that this will work as well

Comment (NejcZdovc) [reply app/common/commonMenu.js]: @sergiorojasa ok this should be working now, can you please check out if everything is ok now? I pushed my fix commit to your PR

**Comment by bsclifton** [comment]
@sergiorojasa congrats on your first merged PR! 😄 👍  Thanks!

**Pull request merged**